### DISCUSSION TITLE
Earler fabric launcher version support

### DISCUSSION THREAD

**Discussion by StillCreative** [OreCruncher/DynamicSurroundingsFabric#6] [Q&A] [answered]
Will you ever make a new version compatible with version 1.11.7 (or earlier)? I'm not able to launch it with my other mods and its disappointing as I've always wanted to have this mod again ever since I had to switch to fabric.

(Edit: wording since I saw the release version is compatible with 1.11.7)

**Comment by OreCruncher** [comment]
There are some features that I use that are not in the 1.11.7 API version.  I'll have to take a look to see if I can make it compatible.  Could result in a feature being disabled if that is the case.  (From what I recall it wasn't anything major.)

**Comment by OreCruncher** [comment, marked as answer]
0.0.4 should have the necessary changes to make it compatible with 0.11.7.
